### PR TITLE
added variants for modal forms

### PR DESCRIPTION
**Pull request by charansuryatejagoud**
Added a 6 variants for Enquiry modal forms & one field for specification tag.

image

image

### CODE REVIEW

File: schemas/ihcl-forms/index.ts [hunk 1]
```diff
@@ -4,24 +4,42 @@ import { FeatureSchemaDefinition } from "schemas/types";
 export const ihclForms: FeatureSchemaDefinition = {
   variants: {
     group: [
-      { title: "[IHCL Forms] Details Form", value: "ihclforms.details-form" },
-      { title: "[IHCL Forms] Payment Form", value: "ihclforms.payment-form" },
+      { title: "[IHCL Forms] Details Form",
+        value: "ihclforms.details-form" 
+      },
+      { title: "[IHCL Forms] Payment Form",
+        value: "ihclforms.payment-form"
```
Comment (NarsingRao13) [inline schemas/ihcl-forms/index.ts:11]: Please use Camelcase for "ihclforms".

File: schemas/ihcl-forms/index.ts [hunk 2]
```diff
@@ -4,24 +4,42 @@ import { FeatureSchemaDefinition } from "schemas/types";
...
+      { title: "[IHCL Forms] Payment Form",
+        value: "ihclforms.payment-form" 
+      },
       {
         title: "[IHCL Forms] Feedback Form",
         value: "ihclforms.group.feedback-form",
       },
-      { title: "[IHCL Forms] Personalize Form", value: "ihclforms.gift-form" },
+      { title: "[IHCL Forms] Personalize Form", 
+        value: "ihclforms.gift-form"
```
Comment (NarsingRao13) [inline schemas/ihcl-forms/index.ts:18]: Same here.

File: schemas/ihcl-forms/index.ts [hunk 3]
```diff
@@ -4,24 +4,42 @@ import { FeatureSchemaDefinition } from "schemas/types";
...
+      {
+        title: "[IHCL Forms] Table Reservation",
+        value: "ihclForms.group.table-reservation",
+      },
     ],
     card: [],
     placeholder: [
-      { title: "[IHCL Forms] Login Form", value: "ihclforms.login-form" },
+      { title: "[IHCL Forms] Login Form",
+        value: "ihclforms.login-form"
```
Comment (NarsingRao13) [inline schemas/ihcl-forms/index.ts:28]: Same here.

File: schemas/ihcl-forms/index.ts [hunk 4]
```diff
@@ -4,24 +4,42 @@ import { FeatureSchemaDefinition } from "schemas/types";
 export const ihclForms: FeatureSchemaDefinition = {
   variants: {
     group: [
-      { title: "[IHCL Forms] Details Form", value: "ihclforms.details-form" },
-      { title: "[IHCL Forms] Payment Form", value: "ihclforms.payment-form" },
+      { title: "[IHCL Forms] Details Form",
+        value: "ihclforms.details-form"
```
Comment (NarsingRao13) [inline schemas/ihcl-forms/index.ts:8]: same here as well

**Comment by NarsingRao13** [comment]
@charansuryatejagoud  please check the nomenclature for all changes. Fix them.

**Comment by charansuryatejagoud** [comment]
Hi @Sukumar ***@***.***>

check this variant you have mention Camel case for "ihclforms"

Thanks & Regards,
Charan Surya,
Contact no 7702094831,
Techouts.

________________________________

**Comment by charansuryatejagoud** [comment]
Change to Camel case is done & added 2 new variants

**Pull request merged**